src/pickup.h

```diff
@@ -17,6 +18,9 @@ class Pickup
         static void do_pickup( const tripoint &pickup_target, bool from_vehicle,
                                std::list<int> &indices, std::list<int> &quantities, bool autopickup );
         static void pick_up( const tripoint &p, int min ); // Pick up items; ',' or via examine()
+        static bool can_pick_one_up( player &c, item &newit, std::string append_fail_msg = "",
```

Comment:
Why is this using non-const player and item references? What properties of the player / item does the function change? If it doesn't change the parameters, they should be const references.